TFT/src/User/API/interfaceCmd.c

```diff
@@ -553,7 +553,7 @@ void sendQueueCmd(void)
             setPrintProgressPercentage(cmd_value());
 
           if (cmd_seen('R'))
-            setPrintRemainingTime((int32_t) (cmd_float() * 60));
+            setPrintRemainingTime((cmd_value() * 60));
```

Comment:
the value in the gcode file is a float not a in (or long int)

Reply:
I checked the values with Prusa sli3r. The values are reported as minutes (int).
![image](https://user-images.githubusercontent.com/22422544/124498191-90af9580-ddd9-11eb-99d1-921fbce25a93.png)

Reply:
this is the gcode file provided to me for testing the M73.
here I see also R1.9 etc...

[Prusa_sample.zip](https://github.com/bigtreetech/BIGTREETECH-TouchScreenFirmware/files/6765447/Prusa_sample.zip)

Reply:
The code in that file is added manually. The slicer generates the parameters in M73 in whole numbers.
Here is a slicer generated file: https://gist.github.com/guruathwal/9b88045e277c3e00a1a87cddbc8c45e3

Reply:
ok, but if I'm not wrong it was reported (maybe by Radek) that it could be set as a fraction of a minute. Maybe someone could also clarify it. just to be sure at 100% it can be only an integer value

Reply:
It is surely reporting as `int`  in Prusa Slicer: 
 https://github.com/prusa3d/PrusaSlicer/blob/3fa78b52b2958be49c442136d8b4971b0a0d4d5c/src/libslic3r/GCode/GCodeProcessor.cpp#L380-L382

Reply:
I didn't talk about the tenth value of switch R ;-)
Test file with decimal value sent by user: belese